*** ninjahelper.in	2012-05-15 15:29:12.000000000 +0400
--- ninjahelper.in.new	2012-05-15 16:04:03.070855690 +0400
***************
*** 27,37 ****
  ##
  require_packages() {
     for pkg in "$@"; do
!       installed=`dpkg -s $pkg | grep 'ok installed'`
        if [ -z "$installed" ]; then
           booleanBox "install $pkg?" "This backup action requires package $pkg. Do you want to install it now?"
           if [ $? = 0 ]; then
!             apt-get install $pkg
              echo "hit return to continue...."
              read
           fi
--- 27,37 ----
  ##
  require_packages() {
     for pkg in "$@"; do
!       installed=`pacman -Qs $pkg | grep $pkg`
        if [ -z "$installed" ]; then
           booleanBox "install $pkg?" "This backup action requires package $pkg. Do you want to install it now?"
           if [ $? = 0 ]; then
!             pacman -S $pkg --noconfirm
              echo "hit return to continue...."
              read
           fi
